### PR TITLE
Update messages when coming back from chat/contact profile.

### LINKED ISSUE #459: group-added-message is not shown at once
when in a chat, the group-member-added-message is not shown immediately. it is shown when leaving a chat and entering it again.

### PR DESCRIPTION
**Pull request by cyBerta**
fixes #459

### CODE REVIEW

File: deltachat-ios/Controller/ChatViewController.swift [hunk 1]
```diff
@@ -195,6 +195,9 @@ class ChatViewController: MessagesViewController {
             }
         }
 
+        // maybe a message was added while the user was in the contact / chat profile
+        refreshMessages()
```
Comment (r10s) [inline deltachat-ios/Controller/ChatViewController.swift]: hm, this loads the message list twice when opening the chat normally, or? one time in here and a second time in viewWillAppear()-> loadFirstMessages().

also, the refreshMessags() does not change the message count.

so, i suggest to use loadFirstMessages()  here and strike the other call for loadFirstMessages()

Comment (r10s) [reply deltachat-ios/Controller/ChatViewController.swift]: did  a commit with the changes while playing/testing, https://github.com/deltachat/deltachat-ios/pull/572 , @cyBerta feel free to merge it to this pr or to close it and do sth else :)

**Comment by cyBerta** [comment]
closing this one in favor of #572

**Comment by r10s** [comment]
as #572 was merged to this pr and not to master, this pr needs to be merged to master as well.

**Pull request merged**